app/logger_config.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from datetime import datetime, timedelta, timezone
+
+IST = timezone(timedelta(hours=5, minutes=30))
+
+
+class ISTFormatter(logging.Formatter):
+    @staticmethod
+    def _converter(*args):
+        return datetime.now(tz=IST).timetuple()
+
+    converter = _converter
+
+
+def setup_logger():
+    logger = logging.getLogger("quizenginelogger")
+
+    logger_format = "%(asctime)s IST loglevel=%(levelname)-6s filename=%(filename)s funcName=%(funcName)s() L%(lineno)-4d %(message)s call_trace=%(pathname)s L%(lineno)-4d"
+
+    formatter = ISTFormatter(fmt=logger_format, datefmt="%Y-%m-%d %H:%M:%S")
+
+    consoleHandler = logging.StreamHandler()
+    consoleHandler.setLevel(logging.DEBUG)
+    consoleHandler.setFormatter(formatter)
```

Comment:
`consoleHandler` doesn't seem to be returned from here. What's it used for?

Reply:
no need to return. We've already added the console handler as a `handler` to the logger. We just need to return the logger, not the handler.